RequestService/RequestService.Core/Services/JobFilteringService.cs

```diff
@@ -73,13 +73,18 @@ public JobFilteringService(IJobService jobService)
 
             if (applyDistanceFilter)
             {
-                jobs = jobs.Where(w => w.DistanceInMiles <= GetSupportDistanceForActivity(w.SupportActivity, distanceInMiles, activitySpecificSupportDistancesInMiles))
-                        .ToList();
+                jobs = jobs.Where(w => w.DistanceInMiles <= GetSupportDistanceForActivity(w.ReferringGroupID, w.SupportActivity, cancellationToken))
+                    .ToList();
             }
 
             return jobs;
         }
 
+        private double? GetSupportDistanceForActivity(int groupId, SupportActivities supportActivity, CancellationToken cancellationToken)
+        {
+            return _groupService.GetGroupSupportActivityRadius(groupId, supportActivity, cancellationToken).Result;
```

Comment:
Better to use await rather than .Result?

I believe the lambda extression above would need to change to

`                jobs = jobs.Where(async (w) => w.DistanceInMiles <= await GetSupportDistanceForActivity(w.ReferringGroupID, w.SupportActivity, cancellationToken))
                    .ToList();
`